chart/env/prod.yaml

```diff
@@ -61,7 +61,8 @@ common:
   hfEndpoint: "https://huggingface.co"
   # Log level
   logLevel: "INFO"
-
+  contentMaxSize: 1000000
```

Comment:
```suggestion
  contentMaxSize: 1_000_000
```

Is it the value we want in production, btw?

Reply:
I fixed it to 10_000_000